containers/kubernetes/reference-content/version-support-policy.mdx

```diff
@@ -18,13 +18,24 @@ categories:
 With Scaleway Kubernetes you can use the latest version of Kubernetes as soon as it is officially released and certified by the [Cloud Native Computing Foundation](https://www.cncf.io/). The versions are usually available in Kubernetes Kapsule and Kubernetes Kosmos within a few days after the release.
 
 <Message type="important">
-  Each minor Kubernetes version is supported by Scaleway Kubernetes for at least twelve months before being deprecated.
+  Scaleway is committed to provide updates on the products lifecycle as follows:
+
+  - Notification of feature deprecation will be announced in the [changelog](/changelog/).
+  - Information regarding a feature's End of Life (EOL) will be communicated through tickets and [scheduled maintenance notices](https://status.scaleway.com/).
+
+  Upon reaching EOL, Scaleway will either remove the feature or initiate a migration (if applicable). While Scaleway endeavors to maintain service continuity, this cannot be assured.
+
+  It is the responsibility of the user to stay informed about both the deprecation (via the changelog) and the EOL (via tickets) of features. Users are also expected to engage with transition plans proactively to prevent any service disruption.
```

Comment:
```suggestion
  It is the responsibility of the user to keep themselves informed about both the deprecation (via the changelog) and the EOL (via tickets) of features. Users are also expected to engage with transition plans proactively to prevent any service disruption.
```